### PR TITLE
Fix currently failing Selenium tests, move to GitHub workflow

### PR DESCRIPTION
**Pull request by mvdbeek**
This fixes the flaky selenium tests that checks for the `my cool annotation` annotation.

### CODE REVIEW

**Comment by mvdbeek** [comment]
First run looks good, I'll retry a couple more times. @dannon when you tested this yesterday, did you test `lib/galaxy_test/selenium/test_workflow_editor.py:WorkflowEditorTestCase.test_collection_input` ? I think the command you pasted in the zoom chat was for a different test.

**Comment by dannon** [comment]
@mvdbeek Yeah, it was the other one I was looking at there (galaxy_test.selenium.test_workflow_editor.WorkflowEditorTestCase.test_non_data_connections), still failing here.

**Review by jmchilton — APPROVED** [review]

**Comment by mvdbeek** [comment]
Hmm, I don't think this is fixing the bug https://jenkins.galaxyproject.org/job/docker-selenium-py3/6962/artifact/6962-test-errors/test_integer_input2020112010501605869402/last.png

**Comment by dannon** [comment]
@mvdbeek Did you try adding a longer selenium timeout after the last keystroke?

**Comment by mvdbeek** [comment]
@galaxybot test this

**Comment by nsoranzo** [comment]
> Run selenium tests w/ pytest, github workflow 

❤️

**Comment by mvdbeek** [comment]
Alright, all the selenium tests are green and done in 38 minutes.

**Comment by jmchilton** [comment]
Amazing 😍😍.

**Review by jmchilton — APPROVED** [review]

File: .github/workflows/selenium.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+          key: pip-cache-${{ matrix.python-version }}-${{ hashFiles('galaxy root/requirements.txt') }}
+      - uses: nanasess/setup-chromedriver@master
+      - name: Run tests
+        run: ./run_tests.sh -selenium lib/galaxy_test/selenium -- --num-shards=3 --shard-id=${{ matrix.chunk }}
+        working-directory: 'galaxy root'
+      - uses: actions/upload-artifact@v2
+        if: failure()
+        with:
+          name: Selenium test results
+          path: 'galaxy root/database/test_errors'
```
Comment (jmchilton) [inline .github/workflows/selenium.yaml:43]: We're losing all the screenshots for the passing PRs right? I put a lot of work into those and I really think they are valuable, but the cost of all the noise and having to maintain a buggy Jenkins setup is probably too high.

Comment (mvdbeek) [reply .github/workflows/selenium.yaml:43]: Yes, I think as an artefact that would probably push us to the limit of what we can store. There is always an option to upload to S3 or similar, if we can get a bit of S3 storage.

**Comment by nsoranzo** [comment]
Awesome, thanks @mvdbeek ! I've disabled `docker-selenium-py3` builds on Jenkins for dev and future release branches.

**Pull request merged**